src/progress-ring/README.md

```diff
@@ -2,12 +2,26 @@
 
 The `vscode-progress-ring` component is used to indicate an indeterminate loading state.
 
-## Attributes
-
-None
+![Progress ring hero](/docs/assets/images/progress-ring-hero.png)
 
 ## Usage
 
+| ❌ Don't                                                                                                           | ✅ Do                                                                                                   |
+| ------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------- |
+| ![Progress ring showing indicating a user progress's in a workflow ](/docs/assets/images/progress-ring-dont-1.png) | ![Image placeholder](/docs/assets/images/progress-ring-do-1.png)                                        |
+| Don't use a Progress Ring to indicate user progress on a multi-step task.                                          | Indicate indeterminate progress while the extension waits for something to load or to finish executing. |
+
+| ❌ Don't                                                                               | ✅ Do                                                                                                                         |
```

Comment:
Similar to the example of multiple rings, do we recommend using adding a progress ring on the parent element if there are multiple items loading?

Reply:
Following up offline—will likely add this in a new PR